test.ts

```diff
@@ -786,3 +786,91 @@ Deno.test("outputToObjectArrayEmpty", function () {
     "Result is not an array or does not have the correct length",
   );
 });
+
+Deno.test("veryLargeAndVerySmall", function () {
+  const db = new DB();
+
+  // return
```

Comment:
Can we remove all of the `// return` section? They effectively only test SQLite code, which should be very well tested by itself 😅

Reply:
Sure, done.